### PR TITLE
Updates package docs 2

### PR DESCRIPTION
**Pull request by graceliching**
Updated for shared schemas and message templates, added new screenshots for current layout

### CODE REVIEW

File: recipe-development-lifecycle.md [hunk 1]
```diff
@@ -4,26 +4,26 @@ date: 2017-05-31 15:00:00 Z
 ---
 
 # Recipe development lifecycle
-Organizations typically plan, develop, test and deploy new integrations via their recipe development lifecycle. This would involve moving recipes from a development environment to a testing environment, or from a testing environment to a production/deployment environment.
+Organizations typically plan, develop, test and deploy new integrations via their recipe development lifecycle. This may involve moving recipes from a development environment to a testing environment, or from a testing environment to a production/deployment environment.
```
Comment (huilinn) [inline recipe-development-lifecycle.md]: Should it be: 
"...via a recipe development lifecycle."?

File: recipe-development-lifecycle.md [hunk 2]
```diff
@@ -4,26 +4,26 @@ date: 2017-05-31 15:00:00 Z
...
 
-On Workato, teams can move sets of recipes and their related assets (e.g. connections, callable recipes, custom connectors) from one environment to another, via the use of packages. This allows teams to maintain their integration recipes across their different teams' environments.
+On Workato, teams can move sets of recipes and their related assets (e.g. connections, custom connectors, lookup tables, callable recipes) from one environment to another, via the use of the recipe development lifecycle. This allows teams to maintain their integration recipes across their different teams' environments.
```
Comment (huilinn) [inline recipe-development-lifecycle.md]: I don't think we can say "...can move sets of recipes...via the use of the lifecycle" - as the lifecycle is a process and not a tool.

File: recipe-development-lifecycle.md [hunk 3]
```diff
@@ -4,26 +4,26 @@ date: 2017-05-31 15:00:00 Z
...
 
-## Packages
-A package refers to a grouping of assets, which refers to recipes and the related dependencies needed for recipes to work, e.g. connections, custom connectors, lookup tables and account properties. With the packages feature, users can move these assets easily across different Workato accounts.
+## Import/Export 
+Users can export and import their recipes as packages. A package refers to a group of assets: recipes and the related dependencies needed for recipes to work, e.g. connections, custom connectors, lookup tables and shared schemas. With this feature, users can move these assets easily across different Workato accounts.
```
Comment (huilinn) [inline recipe-development-lifecycle.md]: Should we specify the contents of a package early here? We cover it under export section, but I think that's the key thing users want to know about packages. Perhaps we can cover it here too or add a link to that section, "here's the list of assets that get exported in a package..."

File: recipe-development-lifecycle.md [hunk 4]
```diff
@@ -34,61 +34,56 @@ When you export a folder of recipes as a zip file, you will find:
...
 - all lookup tables
+- all shared schemas
+- all message templates
 
 To begin exporting your recipes, ensure that all the recipes you wish to export are grouped together in a Workato folder. All associated connections, custom connectors and callable recipes will also be bundled into the package automatically. All account properties and lookup tables in that Workato account are also bundled into the package. Once the package is created, the user can download the package in a zip file.
 
 In the example below, 2 recipes can be found in the **Customer Sync** folder. In these recipes, the user utilizes assets such as callable recipes and lookup tables.
 
-To export a folder as a package, click on the **Export** button. Select the folder to export.
+To export a folder as a package, click on the **Export** button. Select the folder to export. In the second dropdown, select whether lookup table data should be exported in the package. Selecting 'Yes' will allow all the lookup tables and their line data to be exported. If you leave the option as 'No', only the header row in the lookup table will be exported.
```
Comment (huilinn) [inline recipe-development-lifecycle.md]: It's not too clear what the screen is showing here. Should we append this sentence?
"...button." + "You will be brought into an export wizard that guides you through exporting a recipe and its related assets."

Screenshot of screen.

"On this screen, select the folder to export. In the second dropdown..."

File: recipe-development-lifecycle.md [hunk 5]
```diff
@@ -34,61 +34,56 @@ When you export a folder of recipes as a zip file, you will find:
...
-![Export packages - export completed](/assets/images/features/packages/export-packages-complete.png)
+![Export packages - export completed](/assets/images/features/packages/export-packages-complete2.png)
 *Folder has been successfully exported as a package and is ready for download*
 
 ### Importing packages
-Upon clicking the Import/export option on the menu, you'll be brought into the packages page.
 
-![Import export page](/assets/images/features/packages/import-export-buttons.png)
-*Packages page with import and export functions*
+To import a package, click on the 'Import' button on the Import/Export Feed. Find the package to upload in your computer and select a destination Workato folder to import these recipes into. You can also choose whether to import the content of lookup tables.
```
Comment (huilinn) [inline recipe-development-lifecycle.md]: Same comment as above.

"...on the import/export feed." + "You will be brought into an export wizard that guides you through importing a recipe and its related assets."

Screenshot of wizard

"Find the package to upload..."

File: recipe-development-lifecycle.md [hunk 6]
```diff
@@ -34,61 +34,56 @@ When you export a folder of recipes as a zip file, you will find:
...
+| Connections     | Creates a connection placeholder. The user should connect the application after importing. Nothing is done if a connection with the same name already exists.                                                               | Connections     |
+| Lookup tables   | Creates a lookup table with the header rows. Imports the lookup table data if selected during export and import.                                                                                         | Lookup table    |
+| Properties      | Adds properties to existing properties in the Workato account. Does nothing if a property with the same name already exists.                                                                  | Properties      |
+| Shared schemas   | Adds the schema to existing shared schemas in the Workato account or updates it if a schema with the same name already exists.     | Schema reuse |
+| Message templates | Adds the message template to existing templates in the Workato account updates it if a template with the same name already exists. | Message templates |
```
Comment (huilinn) [inline recipe-development-lifecycle.md]: Typo: "...or updates it..."

**Comment by graceliching** [comment]
@huilinn thanks, I've made the changes :)

**Comment by huilinn** [comment]
BTW @graceliching did we minimize the size of the assets? Let's use this https://tinypng.com/

Also, unused images (that you have replaced with latest screenshots) should be deleted.

**Pull request merged**